storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -1695,10 +1695,10 @@ public HeartbeatCache getHeartbeatsCache() {
 
     private Set<List<Integer>> getOrUpdateExecutors(String topoId, StormBase base, Map<String, Object> topoConf,
                                                     StormTopology topology)
-        throws IOException, AuthorizationException, InvalidTopologyException, KeyNotFoundException {
+        throws InvalidTopologyException {
```

Comment:
Can the Nimbus cleanup of parameter (topoId) and throws clause be part of a separate Jira?